lein-cljfmt/src/leiningen/cljfmt.clj

```diff
@@ -105,15 +111,16 @@
      (apply check project (format-paths project))
      (main/abort "No project found and no source paths provided")))
   ([project path & paths]
-   (->> (cons path paths)
-        (transduce
-         (comp (mapcat (partial find-files project))
-               (map (partial check-one project))
-               (map (fn [status]
-                      (print-file-status project status)
-                      (:counts status))))
-         (completing merge-counts))
-        (print-final-count))))
+   (let [counts (->> (cons path paths)
```

Comment:
You could factor out the `->>` part and just put `(cons path paths)` on the end of the transduce.